service/history/shard/context_mock.go

```diff
@@ -632,31 +632,31 @@ func (mr *MockContextMockRecorder) UpdateQueueState(category, state any) *gomock
 }
 
 // UpdateTimerProcessingQueueStates mocks base method.
-func (m *MockContext) UpdateTimerProcessingQueueStates(cluster string, states []*types.ProcessingQueueState) error {
+func (m *MockContext) UpdateTimerProcessingQueueStates(arg0 string, states []*types.ProcessingQueueState) error {
```

Comment:
see, this one changes the first, not the second, while others change the second but not the first.

if it wasn't stable, I'd be concerned, but it re-gens the same every time so...